### PR TITLE
913 update how to branch

### PR DESCRIPTION
**Pull request by Spookyrumble**
## Title

#913 

## Describe your changes: 

Refactor readme to utilize markdown "better". Restructure and rephrase.

## Type of changes:

update how-to-branch readme

## Checklist before requesting a review

* [x] I created a new branch before starting with this ticket?
* [x] I commented the issue after finish in the Frontend Team Board?
* [x] I moved the card to Quality Assurance after finish?

## Did you run into any issues?

- No, issues encountered.

## Screenshots:

- No screenshots to include.

## Feedback

- Yes, I want feedback.

### CODE REVIEW

File: documentation/how-to-branch.md [hunk 1]
```diff
@@ -1,96 +1,85 @@
...
-```
-and then add the file for the listener in that branch. 
-This will give a more sense of ownership, easier for the general overview when you know who has been working on something,
-rather then spending time figure out who has worked on what files. 
+It's recommended for each development team to have its own main working branch (e.g., `teamName-Offers`, `teamName-Listing`). Developers are encouraged to create task-specific branches from their team's main working branch for their individual tasks. However, it's important to consult with the team leads or teachers before proceeding to ensure alignment with the overall project strategy.
 
-![A graphic displaying the branching format](https://i.imgur.com/vdOzdpk.png)
+### Important Notes:
 
+- Avoid working directly in the main/master branch. Always use your team's branch.
```
Comment (Sovak3441) [inline documentation/how-to-branch.md:9]: Main is a protected branch, so it cannot be worked with directly without the proper permissions.
Maby switch up the statement "Avoid work..." with something more reflective about main being protected, and all PR needs minimum 1 review before a merge is accepted ?

**Review by Sovak3441 — CHANGES_REQUESTED** [review]
Nicely formatted.
I'm requesting a review from Abi as well, since this documentation will be in the global scope, and passed on to future classes/teams.

File: documentation/how-to-branch.md (hunk 1 above)
Comment (Spookyrumble) [reply documentation/how-to-branch.md]: Better?

**Review by Sovak3441 — APPROVED** [review]
Much better 👍🏼  10/10

File: documentation/how-to-branch.md [hunk 2]
```diff
@@ -1,96 +1,87 @@
...
 
-ex: 
-Ola Nordmann is working on creating a listener for createOffer. Ola will then create a branch from `FE-Offers` that is named 
-```
-FE-Offers-createOfferListener
-```
-and then add the file for the listener in that branch. 
-This will give a more sense of ownership, easier for the general overview when you know who has been working on something,
-rather then spending time figure out who has worked on what files. 
+It's recommended for each development team to have its own main working branch (e.g., `teamName-Offers`, `teamName-Listing`). Developers are encouraged to create task-specific branches from their team's main working branch for their individual tasks. However, it's important to consult with the team leads or teachers before proceeding to ensure alignment with the overall project strategy.
```
Comment (kyrregjerstad) [inline documentation/how-to-branch.md:5]: Is this the recommended branching strategy we're using now? Not the feature branch strategy?

Comment (Spookyrumble) [reply documentation/how-to-branch.md:5]: I just changed this to be more general, but after thinking about it I think I should perhaps change it to explain how github will give it the issue number if created from the issue..? @Sovak3441 @kyrregjerstad

Comment (kyrregjerstad) [reply documentation/how-to-branch.md:5]: I'm thinking about this though: 

> `It's recommended for each development team to have its own main working branch (e.g., `teamName-Offers`, `teamName-Listing`)`

should we not encourage teams to use feature branches as we are doing now?


```suggestion
Our branching strategy utilizes the `main` branch as the central hub for all development. Developers are required to create new branches off `main` for every feature or bug fix, with the branch name starting with a keyword followed by a forward slash to indicate the type of change, such as feature/, fix/, etc. This is then followed by a description of the purpose, like feature/add-login or fix/header-layout. 

Once development on these branches is complete, a Pull Request (PR) should be initiated to merge the changes back into `main`, ensuring a thorough code review for maintaining quality standards. Branch names should be kept straightforward, avoiding special characters and relying on alphanumeric characters and hyphens for better clarity and compatibility. 
It's crucial for developers to align their work with the overall project strategy before commencing new features or fixes.
```

Comment (Spookyrumble) [reply documentation/how-to-branch.md]: yeah that looks better :)

**Review by kyrregjerstad — APPROVED** [review]

**Comment by AnaCascante** [comment]
LGTM

**Pull request merged**